plugins/permission-common/api-report.md

```diff
@@ -65,6 +75,14 @@ export function isReadPermission(permission: Permission): boolean;
 // @public
 export function isUpdatePermission(permission: Permission): boolean;
 
+// @public
+export type NonEmptyArray<T> = [T, ...T[]];
```

Comment:
Possibly unfortunate to export this. If you mark it with `/** @ignore */` you can avoid exporting it without warnings in the API report. The DX of using the criteria should stay the same afaik, as in you'll see the same editor hints

Reply:
Ah I tried using `@internal` but that broke the api report. I'll give `@ignore` a try!

Reply:
Yeah `@internal` is for when you want to explicitly say that something should not be part of the external API, and API Extractor will then help you enforce that. `@ignore` is for when you wanna tell API Extractor "i know what I'm doing", it's our own invention though.

Update looks 👌